### PR TITLE
Style cancel booking button

### PR DESCRIPTION
**Pull request by Benjamin-GitHub**
+ modifying orange bar just for cells with booking name in

### CODE REVIEW

File: client/src/pages/WeeklyTable.css [hunk 1]
```diff
@@ -52,14 +52,21 @@ th {
 	display: contents;
 }
 
+.visible-on-hover:hover {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
```
Comment (teomrd) [inline client/src/pages/WeeklyTable.css:59]: Do we just need these on hover?

**Review by teomrd — APPROVED** [review]
Just one clarification, otherwise looks good :)

File: client/src/pages/WeeklyTable.css (hunk 1 above)
Comment (Benjamin-GitHub) [reply client/src/pages/WeeklyTable.css:59]: Yes, it's what PO want ;)

Comment (Benjamin-GitHub) [reply client/src/pages/WeeklyTable.css:59]: thanks for your review Theo!

**Pull request merged**